Cargo.toml

```diff
@@ -4,3 +4,7 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
+lazy_static = "1.4"
```

Comment:
Now we have dependencies, there should definitely be a `Cargo.lock` file around here.

Reply:
Huh, this was generated but is ignored by my local git for some reason. -> 69f2455